test/xhr/webfinger.js

```diff
@@ -0,0 +1,23 @@
+define({
+  "status": 200,
+  "responseType": "json",
+  "response": {
+    "subject": "acct:john.joe@example.com",
+    "links":
+    [
+      {
+        "rel" : "okta:idp",
+        "href" : "https://org.okta.com/sso/saml2/0oad5lTSBOMUBOBVVQSC",
+        "titles": {
+          "und" : "Acme Partner IdP"
+        },
+        "properties": {
+          "okta:logo" : "https://ok3static.oktacdn.com/bc/image/fileStoreRecord?id=fs0w8swww6KGUZZWGSHS",
```

Comment:
okta:logo has not implemented yet, does UI need it?

Reply:
no, not yet anyway. I took this example response from the Tech Design just to test